src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import { FC } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: FC<Props> = ({ todo }) => {
+  const { title, user } = todo;
+
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', { 'TodoInfo--completed': todo.completed })}
```

Comment:
You already using destructuring, so use destructured `completed` instead of `todo.completed`

Reply:
thank you, all fixed